### ISSUE TITLE
[Update Request]: Ryujinx.Ryujinx | Ryujinx.Ryujinx.Ava

### ISSUE THREAD

**Issue by Linx-ESP** [microsoft/winget-pkgs#150498]
### What type of update are you requesting?

A new locale, architecture, or installer type of an existing version

### Current Package Identifier

Ryujinx.Ryujinx.Ava

### Package Version

1.1.1287

### Please describe the changes you would like to see

Previously Ryujinx had 2 gui's (GTK and Avalonia), being Avalonia the "experimental" one
Ryujnx now only distributes the Avalonia one and is rename as Ryujinx.exe instead of Ryujinx.ava.exe
This means Ryujinx.Ryujinx.Ava should be removed from WinGet and just keep Ryujinx.Ryujinx

Info at https://github.com/Ryujinx/Ryujinx/wiki/Changelog#111216-111217---2024-03-02

**Comment by vikingnope** [comment]
https://github.com/microsoft/winget-pkgs/pull/150382

**Comment by vikingnope** [comment]
@Exorcism0666 maybe you can take care of this

**Comment by BiosPlus** [comment]
Found this: https://twitter.com/RyujinxEmu/status/1763916049647382644
> It's sure been a while coming but we're happy to announce that our Avalonia frontend is now being rolled out to everyone in version 1.1.1216 onwards! 

Makes sense since if we look at `Ryujinx.exe` and `Ryujinx.Ava.exe` in both the `test-ava-ryujinx-1.1.1291-win_x64.zip` + `ryujinx-1.1.1291-win_x64.zip` files, the SHA256 hash is exactly the same across all 4 (`5CD939BF604F6975113AC7D6CCF237CE9F1BBE07E2A2A80905D256131DC4F059`).

So it's already been live on systems for a while now, the issue may be around if someone has been installing via `Ryujix.Ryujinx.Ava` and winget + the user are just expecting that binary to be replaced. Since the regular manifest references `- RelativeFilePath: publish/Ryujinx.exe` and the AVA manifest references `- RelativeFilePath: publish/Ryujinx.Ava.exe`.

tl;dr: lol, may end up with a lot of uninformed (of the package change) Nintendo fans angry when they realize their emulator isn't updating. /shrug
